arm-templates/ExampleWithProxySQL/masterdb.sql

```diff
@@ -0,0 +1,9 @@
+DROP USER IF EXISTS 'MySQLUSERNAME';
```

Comment:
Add comments to explain the steps. It will be good for others to follow up.

Reply:
I will add the blog link in readme file. I have all explanation there